src/Kunstmaan/NodeBundle/DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,7 @@ public function getConfigTreeBuilder()
                 ->scalarNode('publish_later_stepping')->defaultValue('15')->end()
                 ->scalarNode('unpublish_later_stepping')->defaultValue('15')->end()
                 ->booleanNode('show_add_homepage')->defaultTrue()->end()
+                ->booleanNode('enable_export_page_template')->defaultTrue()->end()
```

Comment:
Maybe set it default to false? This way, default it won't show up on production (or other env's). But when a frontender want to get the base template he can switch that option to `true` just on his version of the site.

What do you think?

Reply:
@acrobat true! changes are done now 😉